projects/valtimo/dashboard-management/src/lib/components/dashboard-management/dashboard-management.component.scss

```diff
@@ -0,0 +1,26 @@
+.valtimo-dashboard-management {
```

Comment:
It feels to me that the no results svg should be moved to the carbon table component, so that it can be reused. It's also used on the dashboard detail page for instance